src/app/main/time-groups/time-schedule-form/time-schedule-form.component.ts

```diff
@@ -28,5 +28,15 @@ export class TimeScheduleFormComponent implements OnInit {
     this.invalidAllDaysTime = this.scheduleDays.filter(day => (day.invalidTime && !day.is_off)).length > 0;
     this.returnedScheduleDays.emit(this.scheduleDays);
   }
+  showCalculateBridge(day: TimeGroupSchedule,index: number){
+    let isShowCountAsBridge;
+    if(index >0 && index <6){
```

Comment:
why you are checking that index >0 and < 6?
I think now index = 6 is not handled

Here Sunday also should have the ability to be counted as a bridge
![Screenshot from 2021-11-23 08-52-56](https://user-images.githubusercontent.com/69426825/142981869-832e1523-6015-4a2d-b918-dbd22c7159fc.png)